### PR TITLE
Routing after signin in

### PR DESCRIPTION
**Pull request by CarolinaCobo**
# ✨ Codu Pull Request 💻

Codu Logo

👉 _**Please remove the below and replace with your own values, leaving the headers where they are.**_ 👈

## Pull Request details:
- If a user is not logged in and tries to bookmark from the article preview page, bookmark/like from within an article will be redirected to the sign-in/login page and then back to where they were. 
- If a user clicks the sign-in button in the navbar will be also redirected back to where they started before signing in. 
- Add a name for the page on the site.manifest
- Add a TODO in the Meta.tsx file for a console error inside Articles.

## Any Breaking changes:
- None

## Associated Screenshots:
    
- None

### CODE REVIEW

File: components/Nav/Nav.tsx [hunk 1]
```diff
@@ -7,13 +7,17 @@ import { Disclosure, Menu, Transition } from "@headlessui/react";
...
 
 function classNames(...classes: string[]) {
   return classes.filter(Boolean).join(" ");
 }
 
 const Nav = () => {
   const { data: session } = useSession();
+  const { route } = useRouter();
+
+  console.log(route);
```
Comment (NiallJoeMaher) [inline components/Nav/Nav.tsx]: You can remove this. 😁

File: pages/get-started/index.tsx [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
-import { useEffect } from "react";
+
 import { LockClosedIcon } from "@heroicons/react/solid";
 
 const GetStarted: NextPage = () => {
-  const { data: session } = useSession();
-  const router = useRouter();
+  const { callbackUrl } = useRouter().query;
+
+  console.log(callbackUrl);
```
Comment (NiallJoeMaher) [inline pages/get-started/index.tsx]: You can remove this. 😁

**Review by NiallJoeMaher — CHANGES_REQUESTED** [review]
Looks good other than the hanging console.logs 🚀

**Review by pkspyder007 — APPROVED** [review]

**Review by NiallJoeMaher — APPROVED** [review]
LGTM!

**Pull request merged**